### ISSUE TITLE
feat(alias): support alias for other version than default

### ISSUE THREAD

**Issue by davinkevin** [shyiko/jabba#388]
Hi, 

I maybe miss something, but I use jabba every day, and switching between v8 and v11 of java with Jabba requires to use the default name of each version (unless you defined one as the default). 

It would be great to be able to do something like this `jabba alias 11 openjdk@1.11.0-1` and then `jabba  use 11`.

**Comment by mikethebos** [comment]
@shyiko @davinkevin Here we go:  [https://github.com/shyiko/jabba/pull/557](https://github.com/shyiko/jabba/pull/557)

**Comment by mikethebos** [comment]
@davinkevin Hey. Can you let us know if my pull request works for you?

**Comment by davinkevin** [comment]
Is there a release made with your PR?

**Comment by mikethebos** [comment]
@davinkevin No, but here is a copy of the binary to replace in .jabba/bin:  https://www.dropbox.com/s/bh9b89s5spzor90/jabba?dl=0

**Comment by mikethebos** [comment]
@davinkevin That is a linux amd64 binary by the way.

**Comment by davinkevin** [comment]
I'm not a huge fan of running a binary distributed like this, for security reason. 

So, to test it, I will have too build a VM and import it...

**Comment by mikethebos** [comment]
@davinkevin or compile my code from my PR... I used go1.13.4.

**Comment by mikethebos** [comment]
@davinkevin @shyiko used go1.11.4, so you may want to compile my PR with that.

**Comment by davinkevin** [comment]
It's perfect for me 👍. You can merge it

**Comment by jborgers** [comment]
Would be a nice improvement! @davinkevin @mikethebos How is this going?

**Comment by mikethebos** [comment]
Works for me.

**Comment by benjamin-rood** [comment]
Should be closed?

**Comment by davinkevin** [comment]
Could be @benjamin-rood, but no release have been made since 2019, so I not able to see it in action sadly…

**Comment by benjamin-rood** [comment]
> Could be @benjamin-rood, but no release has been made since 2019, so I not able to see it in action sadly…

How sad. I think we need to fork this project and take it over if it has been abandoned...

**Comment by willcohen** [comment]
Not abandoned, just a little undermaintained! I’ll try to find a little time tomorrow to work through some backlog. It’s definitely overdue for a release…

**Comment by ZaLiTHkA** [comment]
@willcohen, I'm pretty sure you've already put some thought into this, but might I suggest moving the alias files into a subfolder here..?

for example, creating a new default alias should create a file at `%USERPROFILE%\.jabba\alias\default`, rather than `%USERPROFILE%\.jabba\default.alias`. using the OP example, aliasing a JDK to `8` would then create a file at `%USERPROFILE%\.jabba\alias\8` (which is totally valid, even on Windows).

I don't have any experience with Go, so I can't exactly jump into the code and make this change myself.... but I _suspect_ this might make it easier to work with these alias files deep in Jabba itself.
